spec/lib/publishing_api_finder_publisher_spec.rb

```diff
@@ -2,6 +2,12 @@
 require "publishing_api_finder_publisher"
 
 describe PublishingApiFinderPublisher do
+  before do
+    @schema_type = GovukContentSchemaTestHelpers.configuration.schema_type
+    GovukContentSchemaTestHelpers.configuration.schema_type = "publisher"
```

Comment:
That file is on it's way out in [this PR](https://github.com/alphagov/manuals-publisher/pull/755) 🎉 - so it's just whatever keeps this green 'til then really.